scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.scanner.scanners.fossid
 
 import java.io.File
 import java.io.IOException
+import java.net.Authenticator
```

Comment:
The commit message needs to be adapted to the changes.

Reply:
The commit mess still talks about "credentialsFromNetrc". And while at it, please start capitalized after the ":" in commit message titles.